src/pages/Home/newindex.jsx

```diff
@@ -80,8 +80,8 @@ const NewHome = (props) => {
 										Babybliss voucher allows bearer to shop all products at our online
 										stores and physical locations.
 									</p>
-									<div className="watch-ela span-yellow" onClick={on}>
-										<span className="play span-yellow">
+									<div className="watch-ela span-yellow">
+										<span className="play span-yellow" onClick={on}>
```

Comment:
This should revert. having it in the div covers for both the play icon and text as proposed by the UI
```suggestion
									<div className="watch-ela span-yellow" onClick={on}>
										<span className="play span-yellow">
```

Reply:
Noted